x-pack/plugins/security_solution/public/management/pages/policy/view/event_filters/list/policy_event_filters_list.tsx

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useMemo, useState, useEffect } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiSpacer, EuiText, Pagination } from '@elastic/eui';
+import {
+  ExceptionListItemSchema,
+  FoundExceptionListItemSchema,
+} from '@kbn/securitysolution-io-ts-list-types';
+import { useSearchAssignedEventFilters } from '../hooks';
+import { SearchExceptions } from '../../../../../components/search_exceptions';
+import { useEndpointPoliciesToArtifactPolicies } from '../../../../../components/artifact_entry_card/hooks/use_endpoint_policies_to_artifact_policies';
+import {
+  MANAGEMENT_PAGE_SIZE_OPTIONS,
+  MANAGEMENT_DEFAULT_PAGE_SIZE,
+} from '../../../../../common/constants';
+import { useGetEndpointSpecificPolicies } from '../../../../../services/policies/hooks';
+import {
+  ArtifactCardGrid,
+  ArtifactCardGridProps,
+} from '../../../../../components/artifact_card_grid';
+import {
+  usePolicyDetailsEventFiltersNavigateCallback,
+  usePolicyDetailsSelector,
+} from '../../policy_hooks';
+import { getCurrentArtifactsLocation } from '../../../store/policy_details/selectors';
+import { EventFiltersHttpService } from '../../../../event_filters/service';
+import { useHttp } from '../../../../../../common/lib/kibana';
+
+interface PolicyEventFiltersListProps {
+  policyId: string;
+}
+export const PolicyEventFiltersList = React.memo<PolicyEventFiltersListProps>(({ policyId }) => {
+  const http = useHttp();
+  const eventFiltersService = useMemo(() => new EventFiltersHttpService(http), [http]);
+  const policiesRequest = useGetEndpointSpecificPolicies();
+  const navigateCallback = usePolicyDetailsEventFiltersNavigateCallback();
+  const urlParams = usePolicyDetailsSelector(getCurrentArtifactsLocation);
+  const [expandedItemsMap, setExpandedItemsMap] = useState<Map<string, boolean>>(new Map());
+  const [eventFiltersResults, setEventFiltersResults] = useState<FoundExceptionListItemSchema>();
+
+  const pagination: Pagination = {
+    pageSize: urlParams.page_size || MANAGEMENT_DEFAULT_PAGE_SIZE,
+    pageIndex: urlParams.page_index || 0,
+    pageSizeOptions: [...MANAGEMENT_PAGE_SIZE_OPTIONS],
+    totalItemCount: eventFiltersResults?.total || 0,
+  };
+
+  const { data: eventFilters, isLoading: isLoadingEventFilters } = useSearchAssignedEventFilters(
```

Comment:
someone reading the code here, might think that this hook will return you the policies only assigned to this policyId, but it also returning the global ones.

Reply:
Yeah, same as the comment above. A global artifact is an artifact assigned to all policies.
Imagine we have 3 policies in our system (policy1, policy2, policy3), then
`tags: ['policy:policy1', 'policy:policy2', 'policy:policy3']` === `tags: ['policy:all']`